docfx_project/ziti/overview.md

```diff
@@ -5,9 +5,60 @@ uid: zitiOverview
 # Overview
 
 ## Why Ziti?
-Ziti represents the next generation of secure networking for your applications and has arrived with the Ziti platform,
-created by NetFoundry, Inc ! The Ziti Controller and the Ziti Edge Router combine to form a secure, Zero Trust
-entry point into your network or as a secure zero-trust proxy to other hosted services.
+Ziti represents the next generation of secure, open-source networking for your applications and has arrived with the Ziti platform, created by NetFoundry, Inc ! The Ziti Controller and the Ziti Edge Router combine to form a secure, Zero Trust entry point into your network or as a secure zero-trust proxy to other hosted services.
```

Comment:
I like `developed` vs created.
also not sure what _other_ hosted services means (other compared to what?)

Reply:
also `hosted services` are not defined prior might be confusing